src/mc_rbdyn/Robot.cpp

```diff
@@ -407,6 +407,13 @@ Robot::Robot(NewRobotToken,
     {
       collisionTransforms_[b.name()] = sva::PTransformd::Identity();
     }
+    for(const auto & [body, visuals] : module_._visual)
```

Comment:
Is there any reason to use `_visual` instead of `_collision`?

Reply:
The explanation in https://github.com/jrl-umi3218/mc_rtc/issues/345#issuecomment-1427381490 made sense to me, thanks.